i3_agenda/cache_utils.py

```diff
@@ -32,16 +32,19 @@ def save_cache(events: List[Event]):
         f.write(EventEncoder().encode(events))
 
 
-def get_events_from_cache(f: TextIO) -> List[Event]:
+def get_events_from_cache(f: TextIO):
     events = []
     raw = json.loads(f.read())
     for event in raw:
-        events.append(
-            Event(
-                event["summary"],
-                event["unix_time"],
-                event["end_time"],
-                event["location"],
+        try:
+            events.append(
+                Event(
+                    event["summary"],
+                    event["start_time"],
+                    event["end_time"],
+                    event["location"],
+                )
             )
-        )
+        except KeyError:
+            return None
```

Comment:
I think we should probably delete/overwrite the cache in case it's corrupted (outdated), is it already being done automatically?  
If so, we should probably at least leave a comment in the except clause

Reply:
Yes, I did that cause when I changed the attribute this gave me an KeyError by the dict loaded from the cache. And returning None here, the cache will be reset by the load_events function. I'll put this comments there.

Reply:
Great